spec/usb_webcam/camera_start_capture.yaml

```diff
@@ -0,0 +1,53 @@
+openapi: 3.0.1
+info:
+  title: Generated for EBF-TAS
+  version: '0.2'
+servers:
+- url: https://bfc.timesys.com
+components:
+  securitySchemes:
+    tokenAuth:
+      type: http
+      scheme: token
+security:
+- tokenAuth: []
+paths:
+  /api/v0.2/resources/{ResourceName}/camera/start-capture/{Duration}/:
+    post:
+      tags:
+      - api
+      operationId: api_resources_camera_start_capture
+      parameters:
+      - name: ResourceName
+        in: path
+        description: A unique value identifying this Lab resource.
+        required: true
+        schema:
+          type: string
+      - name: Duration
+        in: path
+        description: Recording duration in seconds.
+        required: true
```

Comment:
I think this field should be optional.  If it is not supplied, the server should use a default recordring duration (maybe 10 seconds).  In labcontrol, I intend to make the default recording duration a server configuration variable (defaulting to 10 seconds).